pre_commit/main.py

```diff
@@ -152,7 +158,10 @@ def main(argv=None):
 
     with error_handler():
         add_logging_handler(args.color)
-        runner = Runner.create()
+        runner_kwargs = {}
+        if hasattr(args, 'config_file'):
+            runner_kwargs['config_file'] = args.config_file
```

Comment:
imo just add this to all the commands (even if they don't use it) (I did this for `--color` recently and it made these sorts of things simpler)